static/docker-compose.yml

```diff
@@ -73,6 +75,12 @@ services:
       PG_CKEDITOR_RW_USER: ${PG_CKEDITOR_RW_USER:?err}
       PG_TENANTS_WRITE_MODE: ${PG_TENANTS_WRITE_MODE:-couchbase_only}
       PG_TENANTS_READ_MODE: ${PG_TENANTS_READ_MODE:-couchbase_only}
+      CKEDITOR_MIGRATE: ${CKEDITOR_MIGRATE}
+      CKEDITOR_SERVER_CONFIG: ${CKEDITOR_SERVER_CONFIG}
```

Comment:
If these don't exist, this will fail. Do you want to provide a blank string if the variable is missing here?